plugins/org.eclipse.n4js.generator.common/src/org/eclipse/n4js/generator/common/GeneratorOption.java

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
There is a class with the exactly the same name `GeneratorOption` in Xtext as well! To avoid name clash, consider use a different name.